10_datastructures_lab/01.py

```diff
@@ -1,19 +1,13 @@
-"""
-Write a program that takes two strings
-from the user and checks if they represent
-a valid user name.
-Valid users and passwords:
-    apple => red
-    lettuce => green
-    lemon => yellow
-    orange => orange
+#!/usr/bin/python
+import os, sys
 
-When valid credentials are entered print:
-    Welcome Master
+userPass = { 'apple' : 'red' , 'lettuce' : 'green' , 'lemon' : 'yellow' , 'orange' : 'orange' }
 
-And when invalid credentials are entered print:
-    INTRUDER ALERT
-
-and exit the program with a non-zero exit code
-"""
+user = raw_input("User: ")
+password = raw_input("Pass: ")
 
+if user in userPass:
+    if userPass[user] == password:
+        print 'Welcome Master'
+    else:
+        print 'Intruder Alert!!'
```

Comment:
1. python developers usually prefer snake_names over camelCase variable names, so user_pass is better than userPass

2. I'd go with this if structure, to show the intruder alert even if user name is incorrect:

```
if user in user_pass and user_pass[user] == password:
  print 'welcome'
else:
  print 'intruder'
```